AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -87,6 +87,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de
 !
 !
+mega.nz##.fm-dialog-overlay
+mega.nz#$#.overlayed .scroll-block { filter: none!important;-webkit-filter: none!important; }
```

Comment:
for such cases rules should be type alike, so both should be `#$#`

also space is missed in second rule before `-webkit...`

Reply:
> for such cases rules should be type alike, so both should be `#$#`

I understand and thus commented, you mean still I should? Then Content blocker etc. will see the cookie consent.

> also space is missed in second rule before `-webkit...`

okay, will change.

Reply:
I mean, we can even remove css filer if you like.

Reply:
Hm, mobile view is ok with those rules. If we use `#$#` for `.fm-dialog-overlay`, it won't work for content blockers.